specificatie/gezag.yaml

```diff
@@ -119,3 +119,9 @@ components:
     GezagNietTeBepalen:
       allOf:
         - $ref: '#/components/schemas/AbstractGezagsrelatie'
+        - type: object
```

Comment:
@CathyDingemanse, @fsamwel, als alleen het gezag veld van meerdere personen wordt gevraagd (RaadpleegMetBurgerservicenummer en ZoekMetAdresseerbaarObjectIdentificatie) en voor meerdere personen kan het gezag niet worden bepaald, dan is het niet duidelijk voor welke personen geen gezag kan worden bepaald. Het opnemen van het minderjarige veld bij het GezagNietTeBepalen type zou dit issue kunnen oplossen. Zijn er bezwaren om dit te doen?

Reply:
Dat lijkt mij een prima plan!

Reply:
> 
> Dit moet dan ook worden toegevoegd aan de gezag-api want daar zie ik het ook niet

@fsamwel, @CathyDingemanse, @FrozenSync, Is dit nog noodzakelijk? Gezag-api levert namelijk altijd bsn van gevraagde persoon en er worden voor een meerderjarige nooit gezagsrelaties geleverd waar de meerderjarige geen relatie mee heeft. Dus ook geen kinderen van de meerderjarige waarvoor geen gezag kan worden bepaald

Reply:
> > Dit moet dan ook worden toegevoegd aan de gezag-api want daar zie ik het ook niet
> 
> @fsamwel, @CathyDingemanse, @FrozenSync, Is dit nog noodzakelijk? Gezag-api levert namelijk altijd bsn van gevraagde persoon en er worden voor een meerderjarige nooit gezagsrelaties geleverd waar de meerderjarige geen relatie mee heeft. Dus ook geen kinderen van de meerderjarige waarvoor geen gezag kan worden bepaald

Dit blijft nog noodzakelijk voor zowel GezagNietTeBepalen en TijdelijkGeenGezag, zoals Frank ook heeft gezegd. Alleen bij GezagNietVanToepassing (deze type bestaat in de rules engine, maar niet in OAS) wordt de gezag object weggelaten. Dit weglaten noemen wij dan "geen gezag".

Reply:
> N.B. niet alleen bij gezag niet te bepalen, ook bij tijdelijk geen gezag wordt er nu geen burgerservicenummer van betreffende minderjarige geleverd.
> 
> Ten behoeve van consistentie met de andere vormen van gezag kan je ervoor kiezen om ook in geval van TijdelijkGeenGezag en GezagNietTeBepalen de minderjarige te leveren.
> 
> Noodzakelijk is het m.i. niet, omdat gezag altijd in de context van een persoon wordt geleverd. Een gebruiker kan dus - zeker wanneer meerdere personen worden gevraagd of verwacht - het burgerservicenummer van de personen ook vragen met fields. Ik denk dat een gebruiker in de praktijk daarnaast ook altijd burgerservicenummer en naam (en evt. ook leeftijd of geboortedatum) van ouders en kinderen zal vragen zodat in één call herkenbare identificerende gegevens van de minderjarige en ouders opgezocht kunnen worden.

Als het in de praktijk niet of nauwelijks voorkomt, zouden we dan niet de regel "alleen implementeren wanneer er naar wordt gevraagd" moeten toepassen?

Reply:
> 
> Ten behoeve van consistentie met de andere vormen van gezag kan je ervoor kiezen om ook in geval van TijdelijkGeenGezag en GezagNietTeBepalen de minderjarige te leveren.
>
Ik dacht dat het in deze p.r. over GezagNietTeBepalen ging. Ik heb dus niet gekeken naar de andere gezag types. Maar om de consistentie door te trekken, waarom wordt er geen toelichting geleverd voor TijdelijkGeenGezag? Willen de afnemers niet weten waarom een minderjarige tijdelijk geen gezag heeft? 

> Noodzakelijk is het m.i. niet, omdat gezag altijd in de context van een persoon wordt geleverd. Een gebruiker kan dus - zeker wanneer meerdere personen worden gevraagd of verwacht - het burgerservicenummer van de personen ook vragen met fields. Ik denk dat een gebruiker in de praktijk daarnaast ook altijd burgerservicenummer en naam (en evt. ook leeftijd of geboortedatum) van ouders en kinderen zal vragen zodat in één call herkenbare identificerende gegevens van de minderjarige en ouders opgezocht kunnen worden.
>
Als in de praktijk altijd bsn wordt gevraagd, waarom wordt deze niet standaard geleverd? Het altijd moeten opgeven van bsn in fields is naar mijn mening (customer zero pet) geen goede developer experience?

Reply:
minderjarige en toelichting velden zijn toegevoegd aan TijdelijkGeenGezag type

Reply:
toelichting bij TijdelijkGeenGezag type wordt op een ander moment gereleased. Dit ivm hogere impact bij het bepalen van correcte toelichting bij tijdelijk geen gezag